post-lib-network.sh

```diff
@@ -46,6 +46,33 @@ function device_ifcfg_key_missing() {
     fi
 }
 
+function check_ifcfg_key_exists() {
```

Comment:
Some comment for the function describing arguments and behavior is really needed here.

Reply:
Truth is that is should be everywhere here. However, this should be done as separate PR I guess.